### DISCUSSION TITLE
Why is the anti importance  different when the random forest is the same?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#336] [General]
library(randomForestSRC)
(rf1 <- rfsrc(mpg ~ ., mtcars, importance = "anti", seed = 11))
(rf2 <- rfsrc(mpg ~ ., mtcars, importance = "anti", seed = 11))
rf1$importance
rf2$importance

the random forest is the same：
![image](https://user-images.githubusercontent.com/5112397/184993617-06f0a10e-f6ab-439b-8ce0-f20078545b38.png)
![image](https://user-images.githubusercontent.com/5112397/184993670-fdbd3d89-3722-4b32-babb-56c4cfc8d99b.png)

but the anti importance is differernt:
![image](https://user-images.githubusercontent.com/5112397/184993750-214e6189-a6e3-47d0-89d0-57cdaa65e777.png)

**Comment by kogalur** [comment]
This issue is discussed in depth in issue #67 (seed does not make variable importance repeatable).

**Comment by A-Pai** [comment]
“The importance="anti" (equivalent to importance=TRUE) assigns cases to the anti(opposite) split.”--this is deterministic，from where the importance randomness comes ？
